android/src/main/java/io/ably/flutter/plugin/AblyMethodCallHandler.java

```diff
@@ -40,31 +40,27 @@
 import io.flutter.plugin.common.MethodChannel;
 
 public class AblyMethodCallHandler implements MethodChannel.MethodCallHandler {
-  private Context applicationContext;
-
-  public interface HotRestartCallback {
-    // Called on every fresh start and on every hot restart
-    void on();
+  public interface ResetAblyClientsCallback {
```

Comment:
I guess we should just ignore how much this looks like [a `Runnable`](https://developer.android.com/reference/java/lang/Runnable), and move on... 😆